bin/generate-shell-css.js

```diff
@@ -0,0 +1,25 @@
+var http = require( 'http' );
+var critical = require( 'critical' );
+
+// Load the HTML page from the dev server. This will fail if the dev server is
```

Comment:
@DanReyLop Yeah I definitely considered that, and couldn't come up with a good way to do it. I think creating this file at build time would be much better than my current solution! 😄 

The problem is that in order to render `index.jade`, we currently need to boot the whole app. I feel like doing this during the build is going to add way too much time...

I suppose we could render the file without the app by using the `jade-cli` manually. We would have stub out the variables that are needed. Might be worth a try? What do you think?